worker/handler/handler.go

```diff
@@ -117,13 +124,32 @@ func NewHandlerManagerSet(opts *config.Config) (hms *HandlerManagerSet, err erro
 		hhits:      &hhits,
 		ihits:      &ihits,
 		misses:     &misses,
+		sbSoftLimit:      opts.Create_sandbox_soft_limit,
+		sbHardLimit:      opts.Create_sandbox_hard_limit,
+		maxNumSbCreating: opts.Max_num_sandbox_creating,
+		numSbCreating:    0,
 	}
+	sbCreatingMutex := &sync.Mutex{}
+	hms.sbCreatingCond = sync.NewCond(sbCreatingMutex)
 
 	hms.lru = NewHandlerLRU(hms, opts.Handler_cache_size) //kb
 
 	return hms, nil
 }
 
+var memUsage float64 = 0.0
+var memLastUpdateTime time.Time = time.Now()
+
+func getMemUsage() float64 {
+	if (time.Since(memLastUpdateTime)) > time.Second {
```

Comment:
Either make this configurable, or add a TODO so we don't forget that we hard-coded the freq to time.Second

Reply:
Would update.